### PR TITLE
Amendments To Add And Delete Sections In UG 

### PR DESCRIPTION
**Pull request by HM33-Stu**
Amendments To Add And Delete Sections In UG With Other Minor Edit(s)

### CODE REVIEW

**Comment by HM33-Stu** [comment]
1 minor edit done based on ystd's consultation notes;
image
*Include app name in Intro heading

File: docs/UserGuide.md [hunk 1]
```diff
@@ -325,54 +325,62 @@ Format: `help`
 
 ### Adding an internship: `add`
 
-Adds an internship entry and all the relevant fields
+With any internship manager, it is important to know how to add an internship entry with all the relevant fields. <br>
```
Comment (HM33-Stu) [inline docs/UserGuide.md:328]: Change to more friendly language use

File: docs/UserGuide.md [hunk 2]
```diff
@@ -325,54 +325,62 @@ Format: `help`
...
-**Common errors:**
-1. If no field prefixes are specified or if there are errors, the command will be rejected with error message:<br>
-   `Invalid Command Prefix!` followed by the format to use and an example for you to refer to or try out
-
 ### Deleting an internship: `delete`
 
-Deletes the internship entry with the corresponding index (based on the displayed list at point of deletion)
+With any internship manager, it is important to know how to delete an internship entry with all the relevant fields. <br>
+
+This command lets you delete the internship entry with the corresponding index (based on the displayed list at point of deletion)
```
Comment (HM33-Stu) [inline docs/UserGuide.md:362]: Change to more friendly language use and also what you should expect from this section

**Comment by HM33-Stu** [comment]
So, Ive converted the try it yourself to point to the tutorial (as discussed) and made it a tip (blue block)
image

**Comment by HM33-Stu** [comment]
I tried my best with the images. The sizes you see are the best I could get out of them such that the image is visible but not too "long". If the image is broad and short, it loses visibility though it looks neater.

image

**Review by jieqiboh — APPROVED** [review]

**Pull request merged**